ftdc/ftdc.go

```diff
@@ -106,9 +107,17 @@ type FTDC struct {
 	//
 	// When debug is true, the `outputWriter` will "tee" data to both the `currOutputFile` and
 	// `inmemBuffer`. Otherwise `outputWriter` will just refer to the `currOutputFile`.
-	debug          bool
-	outputWriter   io.Writer
-	currOutputFile *os.File
+	debug        bool
```

Comment:
Do we ever use the `debug` field (set it to `true`)? My understanding is that tests would pass in their custom `io.Writer` sink with `NewWithWriter` and examine the contents of _that_ buffer, _not_ the `inmemBuffer` used when `debug == true`. Am I missing a case here mentally?

Reply:
I've certainly used it in the past. I believe `debug` predates `NewWithWriter`.

I suspect now that `NewWithWriter` exists, there's little need for a `debug` flag.